### PR TITLE
Bug 1150743 - Update Private browsing advise language

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1150743

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=464100e14282f7db5a970cb8145533060158240d

File: apps/system/locales/system.en-US.properties [hunk 1]
```diff
@@ -602,7 +602,7 @@ share-audio=Share Audio
 
 # Private Browser
 private-window=Private Window
-private-window-detail1=The browser won’t keep any browser history, cookies, or temporary Internet files.
-private-window-detail2=While this device won’t have a record of your browsing history, your Internet service provider can still track the pages you visit.
+private-window-desc1=You have opened a new private window. In private windows, Firefox OS won't keep won't keep any browser history, cookies, or temporary Internet files. However, files you download and bookmarks you make will be kept.
```
Comment (mikehenrty) [inline apps/system/locales/system.en-US.properties]: Remove double `won't keep`.

Comment (KevinGrandon) [inline apps/system/locales/system.en-US.properties]: Nice catch! I even read this a few times, my scumbag brain didn't catch it.

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=356440b0ad4e9f2ea05fcf1cd467cd527359cf1f

**Pull request merged**